translate/src/modules/user/reducer.ts

```diff
@@ -140,12 +142,18 @@ export function reducer(state: UserState = initial, action: Action): UserState {
           notifications: [],
           unread_count: '0',
         },
+        theme: action.data.theme ?? state.theme,
```

Comment:
Can we replace `state.theme` with `''`, to follow the example of other keys?

In any case, we should include `theme` in the output of https://github.com/mozilla/pontoon/blob/main/pontoon/base/views.py#L846.